### ISSUE TITLE
Report Summary is wrong when `--feature-tags=XYZ` is used

### ISSUE THREAD

**Issue by tknerr** [radish-bdd/radish#19]
Given that I have 5 .feature files with 20 scenarios in total
And only 2 .feature files are tagged with "Foo"
When I run `radish *.feature --feature-tags=Foo`
Then report summary should only include the 2 features

But in fact: 
```
$ radish *.feature --feature-tags=Foo

@Foo
Feature: Some Feature of Foo
    Scenario: Foo is not Bar
        ...

@Foo
Feature: Another Feature of Foo
    Scenario: Foo does X
        ...

5 features (2 passed, 0 failed)
20 scenarios (2 passed, 0 failed)
80 steps (10 passed, 0 failed)

Run 1450859378 finished within 4:5.210954 minutes
```

**Comment by tknerr** [comment]
This is probably related to #17

**Comment by timofurrer** [comment]
That's definitely a bug. It'll be fixed in the next version.

Thanks for reporting! :beers:

**Comment by timofurrer** [comment]
Should be fixed fix #f6b842b

**Comment by tknerr** [comment]
Thanks!
